server/plugins/LFUGens.cpp

```diff
@@ -1195,20 +1195,24 @@ void SyncSaw_Ctor(SyncSaw* unit) {
 
 struct K2A : SIMD_Unit {
     ControlRateInput<0> mLevel;
+    ControlRateInput<1> mRamp;
 
     K2A(void) {
         mLevel.init(this);
+        mRamp.init(this);
         if (inRate(0) == calc_ScalarRate)
             set_unrolled_calc_function<K2A, &K2A::next_i<unrolled_64>, &K2A::next_i<unrolled>, &K2A::next_i<scalar>>();
         else
             set_unrolled_calc_function<K2A, &K2A::next_k<unrolled_64>, &K2A::next_k<unrolled>, &K2A::next_k<scalar>>();
     }
 
     template <int type> void next_k(int inNumSamples) {
-        if (mLevel.changed(this))
+        if (mRamp && mLevel.changed(this))
```

Comment:
Should `mRamp` be protected from being negative? (a negative value [would answer true](https://stackoverflow.com/a/18840449/11357554) here, I believe)